### ISSUE TITLE
make wheel with all deps for offline install

### ISSUE THREAD

**Issue by pseudotensor** [h2oai/h2o4gpu#624]
https://medium.com/@amimahloof/how-to-package-a-python-project-with-all-of-its-dependencies-for-offline-install-7eb240b27418

**Comment by pseudotensor** [comment]
assume conda version will suffice. @hemenkapadia

**Comment by hemenkapadia** [comment]
@pseudotensor - it will not. Similar to pip, conda also downloads additional dependencies from the internet. Conda can definitely be leveraged to create a dependency tree and download all the requisite bz2 packages locally. the local directory can then be converted to a channel and be used for offline install. This entire structure can be shipped as a self containing archive if needed.

Do we have any customer asking for this ? Reopening as the self contained offline install need is not being served.

**Comment by pseudotensor** [comment]
I thought that was the original motivation, to be able to install DAI without internet.

**Comment by hemenkapadia** [comment]
Yes, for the customer (citi) that requirement can be met as they have subscription to anaconda enterprise which provides the offline install service as part of that subscription. But only if the packages are available on anaconda.org.

By publishing conda packages we have solved that problem for them (and offloaded the responsibility to help with offline install to anaconda)

If we need a general self contained packed for offline install by any user - then we can use the conda to resolve all the dependencies use that graph to create the self contained artifact, which is what the proposed script will do.
